spec/features/users/sign_up_spec.rb

```diff
@@ -22,6 +22,27 @@
       expect(new_user.last_name).to eq(last_name)
       expect(new_user.email).to eq(email)
     end
+
+    context 'when creating user with the same first_name and last_name' do
+      it 'creates user with correct slug' do
+        first_name = 'Denys'
+        last_name = 'Medynskyi'
+        email = Faker::Internet.email
+
+        FactoryGirl.create(:user, first_name: first_name, last_name: last_name)
+
+        fill_in 'First name', with: first_name
+        fill_in 'Last name', with: last_name
+        fill_in 'Email', with: email
+        fill_in 'Password', with: '123456', match: :prefer_exact
+        fill_in 'Password confirmation', with: '123456', match: :prefer_exact
+
+        expect { click_button 'Sign up' }.to change(User, :count).by(1)
+
+        new_user = User.last
+        expect(new_user.slug).to eq("#{new_user.first_name.downcase}-#{new_user.last_name.downcase}-2")
```

Comment:
`.` is more readable, agree. I don't know anything about SEO, so I will agree on anything)